samples/cellular/modem_shell/sample.yaml

```diff
@@ -10,6 +10,7 @@ tests:
       - nrf9160dk_nrf9160_ns
       - nrf9161dk_nrf9161_ns
       - nrf9151dk_nrf9151_ns
+      - thingy91x_nrf9151_ns
```

Comment:
Let's see how this goes but for 9160 thingy there is a separate build: sample.cellular.modem_shell.thingy91